CHANGES_NEXT_RELEASE

```diff
@@ -36,6 +36,7 @@
 - Fix: invalid regex patterns detection (#968)
 - Fix: Better error returned on invalid geoquery (Issue #2174)
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
+- Add: q-string filters for compound values of attributes (Issue #1156)
```

Comment:
Yes.

What we don't have included is that idea to match 'items in the middle':

E.g.

    q=type==12   would match A1.f.type==12

Not sure I like that feature however